### DISCUSSION TITLE
Doris 中Unique Key模型支不支持物化视图的官方解释是不是存在问题？

### DISCUSSION THREAD

**Discussion by hclou** [apache/doris#35133] [A - General / Q&A]
从文档中两个地方看到了两种不同的说法，那Unique模型是支持物化视图呢，还是不支持呢？

官网文档截图1：
![image](https://github.com/apache/doris/assets/66429730/1940e4a2-c5b8-431a-a985-8958cdc90063)

官网文档截图2：
![image](https://github.com/apache/doris/assets/66429730/f96f2502-fa67-4852-b664-5f445acbe16a)

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
